account_invoice_move_currency/__manifest__.py

```diff
@@ -19,21 +19,22 @@
 ##############################################################################
 {
     'name': 'Account Invoice Move Currency',
-    'version': '9.0.1.1.0',
+    'version': '11.0.1.0.0',
     'author': 'ADHOC SA',
     'category': 'Accounting & Finance',
-    'demo_xml': [],
-    'depends': ['account'],
-    'test': [],
+    'depends': [
+        'account_ux'
```

Comment:
Why account_ux and not account dependency as it was before?

Reply:
@zaoral because the functionality improvement to change invoice currency is in account ux.